WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -95,6 +95,11 @@ final class OrderDetailsViewController: UIViewController {
         super.viewWillAppear(animated)
```

Comment:
@pmusolino – can you verify that this merge is correct?

Reply:
I'm guessing something went wrong in the merge because [CI failed to compile this file](https://app.circleci.com/pipelines/github/woocommerce/woocommerce-ios/12615/workflows/7b898c8a-cbce-4ee6-af4c-3e42226df2c9/jobs/23132):

```
CompileSwift normal x86_64
  /Users/distiller/project/WooCommerce/Classes/ViewRelated/Orders/Order\ Details/OrderDetailsViewController.swift
```